src/Umbraco.Web.UI.Client/src/views/common/overlays/mediaPicker/mediapicker.controller.js

```diff
@@ -133,8 +133,7 @@ angular.module("umbraco")
                     });
                 $scope.currentFolder = folder;
 
-                // for some reason i cannot set cookies with cookieStore
-                document.cookie = "umbLastOpenedMediaNodeId=" + folder.id;
+                localStorageService.set("umbLastOpenedMediaNodeId", folder.id);

```

Comment:
I am not sure what happens here, but if there is a problem, it would also be a problem if a users access got changed, so we should have a look, otherwise I think it really is an edge- case that 2 people share a computer and use the same dialog to pick nodes, and have conflicting access settings. 

@warrenbuckley - to close this, could you check if a wrongly stored node id in localstorage breaks the dialog? - I believe it doesn't, but might as well spend 5 mins on this as we review the dialog anyways.